src/client/doc.h

```diff
@@ -258,4 +258,23 @@ constexpr const char* GmGetR = R"mydelimiter(
             dict
         )mydelimiter";
 
+constexpr const char* GmGetP = R"mydelimiter(
+        get_cga_prints() -> str
+
+        Returns the CGA print operations called during the generation of the model.
+
+        :Returns:
+            str
+        )mydelimiter";
+
+constexpr const char* GmGetE = R"mydelimiter(
+        get_cga_errors() -> str
+
+        Returns the runtime CGA errors and the asset errors occuried during the generation of the model. The asset error 
```

Comment:
"Returns a string with all CGA and asset error messages of this model concatenated by a new line delimiter. The asset error messages additionally contain the key and URI of the asset."